src/components/ListingCard/index.tsx

```diff
@@ -15,24 +17,99 @@ export default function ListingCard({
   isSelected?: boolean;
   onClick?: (id: UUID) => void;
 }) {
+  const [languages, setLanguages] = useState<{ iso_code: string }[]>([]);
+  const [reliefs, setReliefs] = useState<{ relief_code: string }[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const caseLanguages = await getCaseLanguages(caseData.id);
+      const caseReliefs = await getCaseReliefs(caseData.id);
+      setLanguages(caseLanguages);
+      setReliefs(caseReliefs);
+    })();
+  });
```

Comment:
A couple notes, but in short, please remove this code.
This code is dangerous because you're not using a dependency array (the argument after the function, aka callback, to `useEffect`). When you don't pass in a dependency array, React will run this function on **every** render. This, in conjunction with a query call spells large money costs, as, with every render, you make a query call. Imagine this twenty-fold.

Instead of adding a dependency list though, I would suggest using a foreign table query when we query for cases (`getNCases`), which will reduce complexity by querying in one place, and prevent future errors.

Reply:
nice catch that's my bad, I thought I did have the empty dependency array